web/src/beta/features/Editor/index.tsx

```diff
@@ -22,15 +27,23 @@ type Props = {
 };
 
 const Editor: React.FC<Props> = ({ sceneId, projectId, workspaceId, tab, stories }) => {
+  const showDataSourceManager = useReactiveVar(showDataSourceManagerVar);
```

Comment:
I don't know we need this to be global state, do you? If you think so, lets discuss quickly on zoom, since I am currently just doing local state and passing down as props similar state that if there is a good reason to make global maybe i should move those up too.
If not, please just do useState here and then pass down where necessary

Reply:
I'll explain on zoom.